nixos/modules/services/misc/paperless.nix

```
@@ -357,6 +357,14 @@ in
         description = "Settings to pass to the document exporter as CLI arguments.";
       };
     };
+
+    enableExtraFilesOCR = lib.mkOption {
```

```suggestion
    configureTika = lib.mkOption {
```
I would name it like this because the docs mention tika first and only gotenberge as a side service. https://docs.paperless-ngx.com/configuration/#tika